### PR TITLE
Update to latest dev build

### PR DESCRIPTION
**Pull request by ellismg**
In prerpation for 0.15.0-rc1, update to the latest dev versions of
some packages and deal with the breaking changes.

### CODE REVIEW

File: aws/service.ts [hunk 1]
```diff
@@ -275,7 +275,7 @@ function computeImage(
                 if (!registryId) {
                     throw new RunError("Expected registry ID to be defined during push");
                 }
-                const credentials = await aws.ecr.getCredentials({ registryId: registryId });
+                const credentials = await aws.ecr.getCredentials({ registryId: await (<any>registryId).promise() });
```
Comment (ellismg) [inline aws/service.ts]: This is the interesting part of this change, it deals with the breaking change for the shape of data sources arguments. We should feel bad about having to do the downcast from the output here, but it's the quickest way to get this stuff unblocked (and more or less what was happening before).

Comment (lukehoban) [reply aws/service.ts]: Bummer that we have to do this.  I opened https://github.com/pulumi/pulumi-cloud/issues/544 to track cleaning this (and the larger scope of the interaction with the docker buildandpush support here).

Comment (CyrusNajmabadi) [reply aws/service.ts]: i'm looking at this now.

Comment (CyrusNajmabadi) [reply aws/service.ts]: This change is no longer necessary.

**Pull request closed without merging**